applications/nrf_desktop/src/hw_interface/Kconfig

```diff
@@ -184,6 +184,14 @@ config DESKTOP_BATTERY_CHARGER_ENABLE_PIN
 	help
 	  A pin which enables battery charging.
 
+config DESKTOP_BATTERY_CHARGER_ENABLE_INVERSED
+	bool "Inverse charging enable signal"
+	depends on DESKTOP_BATTERY_CHARGER_DISCRETE
+	default n
```

Comment:
"default n" is the default so it is best-practice to omit this line.

Reply:
done